package.json

```diff
@@ -45,10 +45,10 @@
     "@nx/rollup": "16.9.1",
     "@nx/web": "16.9.1",
     "@nx/workspace": "16.9.1",
-    "@openfeature/core": "^0.0.21",
+    "@openfeature/core": "^1.0.0",
     "@openfeature/flagd-core": ">=0.1.10",
-    "@openfeature/server-sdk": "^1.8.0",
-    "@openfeature/web-sdk": "^0.4.0",
+    "@openfeature/server-sdk": "^1.13.2",
+    "@openfeature/web-sdk": "file:.yalc/@openfeature/web-sdk",
```

Comment:
This needs to be updated to `1.0.0` after the web SDK is released, obviously the build will fail until then, but I've tested this locally with `yalc`.